setup.py

```diff
@@ -61,14 +61,15 @@ def is_requirement(line):
     os.system("git push --tags")
     sys.exit()
 
-README = open(os.path.join(os.path.dirname(__file__), "README.rst")).read()
-CHANGELOG = open(os.path.join(os.path.dirname(__file__), "CHANGELOG.rst")).read()
+README = open(os.path.join(os.path.dirname(__file__), "README.rst")).read()  # pylint: disable=consider-using-with
+CHANGELOG = open(os.path.join(os.path.dirname(__file__), "CHANGELOG.rst")).read()  # pylint: disable=consider-using-with
```

Comment:
We could use with instead:
```suggestion
with open(os.path.join(os.path.dirname(__file__), 'README.rst')) as readme:
    README = readme.read()
with open(os.path.join(os.path.dirname(__file__), 'CHANGELOG.rst')) as changelog:
    CHANGELOG = changelog.read()
```

Reply:
Reference: https://github.com/openedx/openedx-events/blob/2de794ed1e83ba7911e71b834b6526f48e4a78fe/setup.py#L65